diff --unified --recursive --text glibc-ports-2.9.orig/sysdeps/arm/preconfigure glibc-ports-2.9/sysdeps/arm/preconfigure
--- glibc-ports-2.9.orig/sysdeps/arm/preconfigure	2005-10-10 11:29:32.000000000 -0400
+++ glibc-ports-2.9/sysdeps/arm/preconfigure	2020-03-29 00:16:51.892818300 -0400
@@ -2,7 +2,7 @@
 arm*)
 	base_machine=arm
 	case $config_os in
-	linux-gnueabi)
+	linux-gnueabi*)
 		machine=arm/eabi/$machine
 		;;
 	*)
